test/parallel/test-file-write-stream.js

```diff
@@ -61,21 +61,12 @@ file
     }
   })
   .on('close', function() {
-    console.error('close!');
     assert.strictEqual(file.bytesWritten, EXPECTED.length * 2);
 
     callbacks.close++;
-    assert.throws(
-      () => {
-        console.error('write after end should not be allowed');
-        file.write('should not work anymore');
-      },
-      {
-        code: 'ERR_STREAM_WRITE_AFTER_END',
-        name: 'Error',
-        message: 'write after end'
-      }
-    );
+    file.write('should not work anymore', common.mustCall((err) => {
+      assert.strictEqual(err.code, 'ERR_STREAM_WRITE_AFTER_END');
+    }));
```

Comment:
The needs to be updated since the timing changes, which is the purpose of the original fix.